tests/monitoring/monitoring.go

```diff
@@ -120,11 +122,9 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 	})
 
 	Context("System Alerts", func() {
-		disableVirtHandler := func() *v1.KubeVirt {
+		disableVirtHandler := func() {
 			originalKv := util.GetCurrentKv(virtClient)
```

Comment:
since now we only use the name and the namespace, can you check if we can avoid this call?

Reply:
so util.GetCurrentKv is calling util.GetKvList(virtClient). I think it's better to leave as is